.changeset/tidy-clocks-sip.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/console": patch
+"@wso2is/i18n": patch
```

Comment:
bump myaccount as well.

Reply:
Any reason to bump the My Account when my i18n changes do not affect the My Account?

Reply:
Added with 9c95fded1e1af7fe035bd0526805f14e82912e24.